packages/template-retail-react-app/app/pages/home/index.jsx

```diff
@@ -285,22 +288,25 @@ const Home = ({productSearchResult, isLoading}) => {
 
 Home.getTemplateName = () => 'home'
 
-Home.shouldGetProps = ({previousLocation, location}) =>
-    !previousLocation || previousLocation.pathname !== location.pathname
+// Home.shouldGetProps = ({previousLocation, location}) =>
+//     !previousLocation || previousLocation.pathname !== location.pathname
 
-Home.getProps = async ({res, api}) => {
+Home.getProps = async ({
```

Comment:
This should be completely removed, are there any blockers that stops you from removing this?

Reply:
You should be able to move 
```
if (res) {
        res.set('Cache-Control', `max-age=${MAX_CACHE_AGE}`)
    }
```

to the component render function.

Reply:
You can call `useServerContext` to get the response object.